docs/ref/forms/fields.txt

```diff
@@ -1054,7 +1054,8 @@ For each field, we describe the default widget used if you don't specify
     * Error message keys: ``required``, ``invalid``
 
     Has the optional arguments ``max_length``, ``min_length``, and
-    ``empty_value`` which work just as they do for :class:`CharField`.
+    ``empty_value`` which work just as they do for :class:`CharField`,
+    and ``assume_scheme`` that defaults to ``"http"``.
```

Comment:
We should add two annotations:
```
    .. versionchanged:: 5.0

        The ``assume_scheme`` argument was added.

    .. deprecated:: 5.0

        The default value for ``assume_scheme`` will change from ``"http"`` to
        ``"https"`` in Django 6.0.    
```

Reply:
We should also add release notes and a note in `docs/internals/deprecation.txt`, e.g.

```diff
diff --git a/docs/releases/5.0.txt b/docs/releases/5.0.txt
index 84689f223d..079de61376 100644
--- a/docs/releases/5.0.txt
+++ b/docs/releases/5.0.txt
@@ -151,7 +151,8 @@ File Uploads
 Forms
 ~~~~~
 
-* ...
+* The new ``assume_scheme`` argument for :class:`~django.forms.URLField` allows
+  specifying a default URL scheme.
 
 Generic Views
 ~~~~~~~~~~~~~
@@ -281,6 +282,9 @@ Miscellaneous
   :class:`~django.db.models.BaseConstraint` is deprecated in favor of
   keyword-only arguments.
 
+* The default scheme for ``forms.URLField`` will change from ``"http"`` to
+  ``"https"`` in Django 6.0.
+
 Features removed in 5.0
 =======================
 
```

```diff
diff --git a/docs/internals/deprecation.txt b/docs/internals/deprecation.txt
index b151020375..622dc5f359 100644
--- a/docs/internals/deprecation.txt
+++ b/docs/internals/deprecation.txt
@@ -21,6 +21,9 @@ details on these changes.
 * Support for passing positional arguments to ``BaseConstraint`` will be
   removed.
 
+* The default scheme for ``forms.URLField`` will change from ``"http"`` to
+  ``"https"``.
+
 .. _deprecation-removed-in-5.1:
 
 5.1
```